frontend/mr-admin-flate/src/components/notifikasjoner/Notifikasjonsbjelle.tsx

```diff
@@ -10,16 +10,14 @@ function Notifier() {
 
 export function Notifikasjonsbjelle() {
   const { data: features } = useFeatureToggles();
-  const {
-    data: antallUlesteNotifikasjoner,
-    isLoading: isLoadingUlesteNotifikasjoner,
-  } = useNotificationSummary();
+  const { data: summary, isLoading: isLoadingUlesteNotifikasjoner } =
+    useNotificationSummary();
 
-  if (isLoadingUlesteNotifikasjoner && !antallUlesteNotifikasjoner) {
+  if (isLoadingUlesteNotifikasjoner || !summary) {
```

Comment:
Endret på logikk for å slippe nullable kode under - blir det fortsatt riktig?

Reply:
Tenker det går fint ja